.github/workflows/release.yml

```diff
@@ -1,9 +1,6 @@
 name: Build & Release
 
-on:
-  push:
-    branches:
-      - main
+on: workflow_dispatch
```

Comment:
Why manual release? Why not run it together with a new pushed tag?

Reply:
That was the original plan, but then I realized that changes to stuff like the runner script don't affect the resulting package so there is no reason to create new releases in that case. However, it's not unlikely that we will automate this somehow in the future.

Reply:
Ah, I think I misunderstood. Triggering on tag pushed could work too, but then we shift the requirement of ensuring that the tagged commit builds and passes tests successfully to the dev pushing the tag.